tdms/include/field.h

```diff
@@ -49,8 +49,9 @@ class Grid {
   /**
    * @brief Set the preferred interpolation methods
    */
-  void set_preferred_interpolation_methods(PreferredInterpolationMethods _pim) {
-    pim = _pim;
+  void
+  set_preferred_interpolation_methods(tdms_flags::InterpolationMethod _pim) {
```

Comment:
```suggestion
  void  set_preferred_interpolation_methods(tdms_flags::InterpolationMethod _pim) {
```
Did clang-format do this? The newline makes codecov complain, because +1 LoC.